Packages/com.unity.inputsystem/InputSystem/InputManager.cs

```diff
@@ -1207,6 +1207,16 @@ public InputDevice AddDevice(InputDeviceDescription description)
             return device;
         }
 
+        public InputDevice AddDevice(InputDeviceDescription description, InternedString layout, string deviceName = null,
+            int deviceId = InputDevice.InvalidDeviceId, InputDevice.DeviceFlags deviceFlags = 0)
```

Comment:
Out of curiosity, is null / InvalidDeviceId a reasonable default value(s), e.g. adding a device that is not valid is the "common" operations?

Reply:
I don't know if it's reasonable, but it's consistent :smile:

Reply:
(only exception: for any kind of device, we take IDs across domain reloads; so in that case, no new IDs should be generated)

Reply:
Allright, thanks for taking your time to explain the why, all good with me then